lib/doorkeeper/oauth/authorization/uri_builder.rb

```diff
@@ -20,7 +20,7 @@ def uri_with_fragment(url, parameters = {})
         end
 
         def build_query(parameters = {})
-          parameters = parameters.reject { |k, v| v.blank? }
+          parameters = parameters.reject { |_k, v| v.blank? }
```

Comment:
> Yeap, you are right. Аs well as for https://github.com/doorkeeper-gem/doorkeeper/blob/master/lib/doorkeeper/config.rb#L182

In that case I like the documentation `_routes` provide. Here, I thought `_k` for key wasn't necessary (or I'd expand it to `_key, value`.